chromadb/test/property/test_add.py

```diff
@@ -2,15 +2,14 @@
 from hypothesis import given
 import chromadb
 from chromadb.api import API
-from chromadb.test.configurations import configurations
+from chromadb.test.fixtures import fixtures
 import chromadb.test.property.strategies as strategies
 import chromadb.test.property.invariants as invariants
 
 
-@pytest.fixture(scope="module", params=configurations())
+@pytest.fixture(scope="module", params=fixtures())
```

Comment:
See if this can be done with stacked/dependent fixtures.

Reply:
No, params are the right construct here because we want all tests with this fixture to run multiple times, once for each param.